examples/svelte-kit-example/src/routes/sign-up/+page.svelte

```diff
@@ -22,6 +27,10 @@ const handleSignUp = async () => {
 		name: `${user.firstName} ${user.lastName}`,
 		callbackURL: "/",
 		fetchOptions: {
+			onSuccess() {
+				alert("Your account has been created.");
```

Comment:
Because I'm not a native English speaker, a native check is wanted if possible.

Reply:
Not on the core team, but this is perfect.

Reply:
@tehnrd Thanks for the review!